subsys/bluetooth/services/Kconfig.bms_srv

```
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig BT_GATT_BMS_SRV
+	bool "Bond Management Service"
+	select BT_NRF_SERVICES
+	depends on !BT_SMP_ALLOW_UNAUTH_OVERWRITE
```

Sorry, I should have looked up the correct name I wanted.
```suggestion
	depends on !BT_KEYS_OVERWRITE_OLDEST
```

---

fixed